source/keyLabels.py

```diff
@@ -162,3 +162,6 @@
 	# Translators: This is the name of a key on the keyboard.
 	'tab': pgettext("keyLabel", "tab"),
 }
+
+def getKeyCombinationLabel(keyCombination):

```

Comment:
Please add a docstring explaining what this does. It need only be one or two lines.